compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -113,10 +113,10 @@ public void stage1_CreateHyperdiskStoragePoolTest()
     Assert.assertEquals(10240, storagePool.getPoolProvisionedCapacityGb());
     Assert.assertTrue(storagePool.getStoragePoolType().contains("hyperdisk-balanced"));
     Assert.assertTrue(storagePool.getCapacityProvisioningType().equalsIgnoreCase("advanced"));
+    Assert.assertTrue(storagePool.getPerformanceProvisioningType().equalsIgnoreCase("advanced"));
```

Comment:
please use `PERFORMANCE_PROVISIONING_TYPE` instead of `"advanced"` literal

Reply:
Thank you. Done